README.md

```diff
@@ -285,6 +285,18 @@ resp, err := rmqc.DeleteShovel("/", "a.shovel")
 
 ```
 
+### Operations on cluster name
+``` go
+// Get cluster name
+cn, err := rmqc.GetClusterName()
+// => ClusterName, err
+
+// Rename cluster
+resp, err := rmqc.SetClusterName(ClusterName{Name: "rabbitmq@rabbit-hole"})
```

Comment:
@gerhard and I discussed this and perhaps using a struct is not too odd. It would be more flexible should this part of the API change or require validation and it's not too weird for Go libraries to do something like this.

So no need for changing it. Thank you for responding so quickly!